index.js

```diff
@@ -1,10 +1,9 @@
 'use strict';
 
 let fetch = require('node-fetch');
-
 let BotWrapper = require('./lib/BotWrapper.js');
 let ftl = require('./lib/ftl.js');
-
+let diceRoller = require('./commandScripts/diceRoller.js');
```

Comment:
This path is probably going to show up a lot, so there is reason to think about it. I think it should be something shorter (like `commands/diceRoller.js`). What does the word `Scripts` add to the meaning of this name?